functions/__fzf_search_shell_variables.fish

```diff
@@ -10,11 +10,16 @@ function __fzf_search_shell_variables --description "Search and inspect shell va
     # __echo_value_or_print_message will print an informative message in lieu of the value.
     set variable_name (
         set --names |
-        fzf --preview '__fzf_display_value_or_error {}'
+        fzf --preview '__fzf_display_value_or_error {}' \
+            --query=(commandline --current-token | read token && string replace '$' '' $token)
```

Comment:
Can' you just do 
```
--query=(commandline --current-token | string replace '$' '' $token)
```?

Reply:
I was thinking about saving to call `commanline --current-token` twice, but now I think about it again, it's pretty cheap.